main/template/default/auth/sessions_catalog.tpl

```diff
@@ -87,65 +87,70 @@
+                                <button class="btn" type="submit">{{ 'Search' | get_lang }}</button>
+                            </div>
+                        </form>
+                    </li>
                 </ul>
             </div>
         {% endif %}
     </div>
     <div class="span9">
-               {% for session in sessions_blocks %}
-                <div class="well well-small session-group" id="session-{{ session.id }}">
-                    <div class="row-fluid">
-                        <div class="span9">
-                            <div class="row-fluid padding-clear">
-                                <div class="span2">
-                                    <span class="thumbnail">
-                                        {{ session.icon }}
-                                    </span>
-                                </div>
-                                <div class="span10 border-info">
-                                    <h3>{{ session.name }}</h3>
-                                    <div class="tutor"><img src="{{ _p.web }}/main/img/teachers.gif" width="16px"> {{ 'GeneralCoach' | get_lang }} {{ session.coach_name }}</div>
-                                </div>
+        {% for session in sessions_blocks %}
+            <div class="well well-small session-group" id="session-{{ session.id }}">
+                <div class="row-fluid">
+                    <div class="span9">
+                        <div class="row-fluid padding-clear">
+                            <div class="span2">
+                                <span class="thumbnail">
+                                    {{ session.icon }}
+                                </span>
                             </div>
-                            <div class="row-fluid">
-                                <div class="accordion" id="session-{{ session.id }}-accordion">
-                                    <div class="accordion-group">
-                                        <div class="accordion-heading">
-                                             <a class="accordion-toggle" data-toggle="collapse" data-parent="#session-{{ session.id }}-accordion" href="#session-{{ session.id }}-courses">
-                                                {{ 'CourseList' | get_lang }}
-                                            </a>
-                                        </div>
-                                        <div id="session-{{ session.id }}-courses" class="accordion-body collapse in">
-                                            <div class="accordion-inner"></div>
-                                        </div>
+                            <div class="span10 border-info">
+                                <h3>{{ session.name }}</h3>
+                                <div class="tutor"><img src="{{ _p.web }}/main/img/teachers.gif" width="16px"> {{ 'GeneralCoach' | get_lang }} {{ session.coach_name }}</div>
```

Comment:
Avoid .gif icons. Prefer .png versions in main/img/icons/. Maybe there is a returnIcpnPath modifier? There should be one, in any case.